src/utilities/effects.ts

```diff
@@ -1143,8 +1145,15 @@ const effects = {
           showSuccessToast(`Scheduling ${analysis_only ? 'Analysis ' : ''}Complete`);
         } else if (status === 'failed') {
           schedulingStatus.set(Status.Failed);
-          console.log(reason);
+
```

Comment:
I think we also need to `schedulingErrors.set([])` when we run schedule in case a previous run had a failure but the current run succeeds.

Reply:
good point. what about when the status is "incomplete"? does that have an error associated with that?

Reply:
Nope it should not